### ISSUE TITLE
Speakser Service TypeError

### ISSUE THREAD

**Issue by jnbn** [ebaauw/homebridge-zp#189]
#### Issue
Getting a typescript error when I select Speaker as a service which leads an error and I can't use any speakers. Switching back to switch works as expected.

#### Log Messages
```
[12/4/2021, 4:43:46 PM] [Sonos] warning: TypeError: Invalid Version: 12.1
    at new SemVer (/usr/local/lib/node_modules/homebridge-zp/node_modules/semver/classes/semver.js:38:13)
    at Object.major (/usr/local/lib/node_modules/homebridge-zp/node_modules/semver/functions/major.js:2:29)
    at SystemInfo.getMacOsInfo (/usr/local/lib/node_modules/homebridge-zp/node_modules/homebridge-lib/lib/SystemInfo.js:352:20)
    at SystemInfo.init (/usr/local/lib/node_modules/homebridge-zp/node_modules/homebridge-lib/lib/SystemInfo.js:181:25)
    at ZpPlatform._main (/usr/local/lib/node_modules/homebridge-zp/node_modules/homebridge-lib/lib/Platform.js:222:7)
```

**Comment by ebaauw** [comment]
Did you remove the cached accessories when changing the service?

**Comment by ebaauw** [comment]
Seeing the TypeError myself as well, on macOS 12.1.

**Comment by jnbn** [comment]
Just saw your previous message sorry :) Seems you've already taken action. Thanks

**Comment by ebaauw** [comment]
Fixed in homebridge-lib v5.1.22.  If you re-install Homebridge ZP, it should pick up this version.
